application/src/port.rs

```diff
@@ -1,17 +1,17 @@
-pub use crate::projection::TodoListProjection;
-pub use domain::{todolist_event::TodoListEvent, todolist_state::TodoList};
+pub use crate::projection::TodoList;
+pub use domain::todolist_event::TodoListEvent;
 use framework::*;
 
 #[async_trait]
 #[delegate]
 pub trait TodoListStore {
-    async fn pull(&self) -> AnyResult<TodoList>;
+    async fn pull(&self) -> AnyResult<Vec<TodoListEvent>>;
     async fn push(&self, events: &[TodoListEvent]) -> AnyResult<()>;
```

Comment:
**Suggestion:** Consider renaming `pull` and `push` methods to more descriptive names like `get_events` and `store_events` respectively. [enhancement]
```suggestion
pub trait TodoListStore {
    async fn get_events(&self) -> AnyResult<Vec<TodoListEvent>>;
    async fn store_events(&self, events: &[TodoListEvent]) -> AnyResult<()>;
}
```